jetty-io/src/main/java/org/eclipse/jetty/io/RetainableByteBuffer.java

```diff
@@ -27,20 +28,20 @@
  */
 public class RetainableByteBuffer implements Retainable
 {
```

Comment:
javadoc above is now wrong.  No `ByteBufferPool` is passed.

Reply:
Fixed.